### PR TITLE
chore: run gh pages jobs sequentially

### PR DESCRIPTION
**Pull request by imGok**

### CODE REVIEW

**Comment by hoangdat** [comment]
@tk-nguyen please reivew

**Comment by tk-nguyen** [comment]
We don't need to. This action when they release a new version will push to v3 tag

File: .github/workflows/gh-pages.yaml [hunk 1]
```diff
@@ -65,7 +67,7 @@ jobs:
           echo "URL=https://$GITHUB_REPOSITORY_OWNER.github.io/${GITHUB_REPOSITORY##*/}/$FOLDER" >> $GITHUB_OUTPUT
 
       - name: Deploy to GitHub Pages
-        uses: peaceiris/actions-gh-pages@v3
+        uses: peaceiris/actions-gh-pages@v3.9.3
```
Comment (tk-nguyen) [inline .github/workflows/gh-pages.yaml:70]: We don't need to. This action when they release a new version will push to v3 tag

Comment (imGok) [reply .github/workflows/gh-pages.yaml:70]: ok, I'm trying to find why we have 404 on every ghpages deployed. do you have any idea?

Comment (tk-nguyen) [reply .github/workflows/gh-pages.yaml:70]: It need to finish the 2nd action to deploy a new folder: https://github.com/linagora/twake-on-matrix/actions/runs/6417229541

If too many PRs are opened this action will be cancelled, so some PR might not have it

Comment (imGok) [reply .github/workflows/gh-pages.yaml:70]: oh ok I get it tyvm

File: .github/workflows/gh-pages.yaml [hunk 2]
```diff
@@ -11,6 +11,8 @@ jobs:
   deploy_preview:
     name: Deploy preview versions on pull requests
     runs-on: ubuntu-latest
+    concurrency:
```
Comment (tk-nguyen) [inline .github/workflows/gh-pages.yaml:14]: We can still merge this part.

File: .github/workflows/gh-pages.yaml [hunk 3]
```diff
@@ -11,6 +11,8 @@ jobs:
...
+      group: ${{ github.workflow }}-${{ github.ref }}
```
Comment (tk-nguyen) [inline .github/workflows/gh-pages.yaml:15]: We can still merge this part.

Comment (imGok) [reply .github/workflows/gh-pages.yaml:15]: yes, it's to avoid `The process '/usr/bin/git' failed with exit code 1`

Comment (tk-nguyen) [reply .github/workflows/gh-pages.yaml:15]: Ok, please push a new commit to remove the tag change above, then I'll merge this.

Comment (imGok) [reply .github/workflows/gh-pages.yaml:15]: updated on my side

**Review by tk-nguyen — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**